packages/xrpl/src/models/transactions/checkCash.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unnecessary-condition -- JavaScript users can always pass in undefined */
```

Comment:
Is this only called in one place? If so, it should just disable that one line

Reply:
There were two, but they were close together so I did a disable + enable pair to cover them more closely.